### DISCUSSION TITLE
Timestamp workflow not working, issue with Metadata aqquisition?

### DISCUSSION THREAD

**Discussion by AshhadQazi** [bonsai-rx/bonsai#1861] [Q&A] [answered]
Hello!

[Saving Timestamp.zip](https://github.com/user-attachments/files/15964361/Saving.Timestamp.zip)

I am reaching out for assistance with a Bonsai file I have been working on, aimed at obtaining timestamps for video footage. Attached to this email, you will find the Bonsai file along with an example of the preferred output format when writing into a CSV. For consistency in future data analysis, it would be ideal to maintain this format. However, the primary challenge is getting the file to function correctly.
![image](https://github.com/bonsai-rx/bonsai/assets/170653199/e85f193f-3a09-4247-8fc0-9777e8a4208b)

While I am aware that the Spinnaker Capture node has its metadata available, this workflow is older, and I have only recently started working with it. I have explored the member selector to identify a possible substitution but have been unable to find one.

Does anyone have any suggestions on how to resolve this issue? Could it be related to the acquisition process, or perhaps an invalid selector?

Thank you in advance for your assistance.

Thank you

**Comment by bruno-f-cruz** [comment, marked as answer]
The class from their API changed name. It is now `ChunkData`:

![image](https://github.com/bonsai-rx/bonsai/assets/7049351/1174a00e-6d97-42a2-992e-bb5f764de2f0)
